pkg/packagekit/autoupdates.jsx

```diff
@@ -59,7 +59,7 @@ class ImplBase {
 
 class DnfImpl extends ImplBase {
     getConfig() {
-        var dfd = new cockpit.defer();
+        let dfd = cockpit.defer();
```

Comment:
nitpicking: this can be `const`

Reply:
Indeed the `dfd` reference doesn't change, so technically it could be `const`. However, I still don't actually like to think in that way: a promise state changes, so in my mind it shouldn't be a constant.